src/components/Quiz/Quiz.js

```diff
@@ -4,111 +4,132 @@ import { Link } from 'react-router-dom';
 
 const Quiz = ({ questions, checkAnswer, score, resetScore }) => {
   const [ userAnswers, setUserAnswers ] = useState([])
-  const [ step, setStep ] = useState(0)
+  const [ questionNumber, setQuestionNumber ] = useState(0)
   const [ buttonDisabled, setButtonDisabled ] = useState(true)
   const [ formDisabled, setFormDisabled ] = useState(false)
 
   const handleChange = (answer) => {
+    console.log(answer)
     setFormDisabled(true);
     setButtonDisabled(false);
     setUserAnswers([ ...userAnswers, answer ])
-    checkAnswer(answer, step)
+    checkAnswer(answer, questionNumber)
   }
 
   const changeQuestion = (event) => {
     event.preventDefault()
 
-    if (event.target.value === 'next' && userAnswers[step+1] === undefined) {
+    if (event.target.parentNode.value === 'next' && userAnswers[questionNumber + 1] === undefined) {
       setFormDisabled(false);
       setButtonDisabled(true);
-      setStep(step + 1)
-    } else if (event.target.value === 'next' && userAnswers[step+1] !== undefined) {
+      setQuestionNumber(questionNumber + 1)
+    } else if (event.target.parentNode.value === 'next' && userAnswers[questionNumber + 1] !== undefined) {
       setFormDisabled(true);
       setButtonDisabled(false);
-      setStep(step + 1)
-    } else if (event.target.value === 'previous') {
-      setStep(step - 1)
+      setQuestionNumber(questionNumber + 1)
+    } else if (event.target.parentNode.value === 'previous') {
+      setQuestionNumber(questionNumber - 1)
       setFormDisabled(true);
       setButtonDisabled(false);
     }
   }
 
   const showResult = () => {
-    if (userAnswers[step] !== undefined && userAnswers[step] === questions[step].correctAnswer) {
+    if (userAnswers[questionNumber] !== undefined && userAnswers[questionNumber] === questions[questionNumber].correctAnswer) {
       return (
-        <>
-          <p>✅ Awesome job!</p>
-          <p>{questions[step].fact}</p>
-        </>
+        <div className="fact-container">
+          <p className="result">✅ Awesome job!</p>
+          <p>{questions[questionNumber].fact}</p>
+        </div>
       )
-    } else if (userAnswers[step] !== undefined && userAnswers[step] !== questions[step].correctAnswer) {
+    } else if (userAnswers[questionNumber] !== undefined && userAnswers[questionNumber] !== questions[questionNumber].correctAnswer) {
       return (
-        <>
-          <p>❌ Let’s look at that one again!</p>
-          <p>{questions[step].fact}</p>
-        </>
+        <div className="fact-container">
+          <p className="result">❌ Let’s look at that one again!</p>
```

Comment:
I like the "let's look at that one again" language.  Much better than saying "wrong!" and it redirects the focus back to the point that this is about learning for the app user.